### PR TITLE
[BOYSCOUT] Factorize common styles

### PR DESCRIPTION
**Pull request by Sevrahk**

### CODE REVIEW

File: src/searchForm/baseStyles.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { space, font, color, radius } from '_utils/branding'
+
+const overlayWidth = '375px'
```
Comment (Tuxosaurus) [inline src/searchForm/baseStyles.tsx]: nit: not sure if this should be placed in `componentSizes` in `branding`

File: src/searchForm/baseStyles.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+export const overlayBaseStyle = `
+  box-sizing: border-box;
+  width: ${overlayWidth};
+  border: 1px solid ${color.border};
+  border-radius: ${radius.l};
+  padding: ${space.xl};
+  padding-top: ${space.s};
+  /* Same as TripCard, we might need to have box-shadow specs so that we can move it to branding file */
+  box-shadow: 0 1pt 4pt rgba(0, 0, 0, 0.16), 0 2pt 8pt rgba(0, 0, 0, 0.08);
```
Comment (wakooka) [inline src/searchForm/baseStyles.tsx]: Maybe just move this to branding file.

File: src/searchForm/stepper/overlay/index.tsx [hunk 3]
```diff
@@ -1,21 +1,11 @@
...
-    box-sizing: border-box;
-    width: ${overlayWidth};
-    border: 1px solid ${color.border};
-    border-radius: ${radius.l};
-    padding: ${space.xl};
-    padding-top: ${space.s};
-    /* Same as TripCard, we might need to have box-shadow specs so that we can move it to branding file */
-    box-shadow: 0 1pt 4pt rgba(0, 0, 0, 0.16), 0 2pt 8pt rgba(0, 0, 0, 0.08);
-    background: ${color.defaultBackground};
+    ${overlayBaseStyle}
```
Comment (gabrielagabriel) [inline src/searchForm/stepper/overlay/index.tsx]: question: if you are already overriding the component SepperOverlay why do you need to use overlayBaseStyleHere?

Comment (Sevrahk) [reply src/searchForm/stepper/overlay/index.tsx]: I don't override components I just factorized common elements and I call them in components ;)

Comment (gabrielagabriel) [reply src/searchForm/stepper/overlay/index.tsx]: I meant ```const StyledComponent = styled(OtherComponent)``` as an override.
The question is that you could already apply this style to that component directly.. I guess not.

File: src/searchForm/baseStyles.tsx (hunk 2 above)
Comment (Sevrahk) [reply src/searchForm/baseStyles.tsx]: I directly used `shadow.searchForm` since we already defined it :)

**Pull request merged**